test/integration/userFlow.test.js

```diff
@@ -16,6 +16,7 @@ afterAll(async function afterAllTests() {
 const contextNoToken = global.mockedContextWithOutToken;
 const invalidContext = global.mockedContextWithInValidToken;
 const validContextCharlie = global.mockedContextWithValidTokenForCharlie;
+const noEmailContextMarv = global.mockedContextWithNoEmailTokenForMarv;
```

Comment:
could we please use contextNoEmail instead? I think using names in constants is fine if we test a scenario where we validate behaviour with multiple users but this only tests the lack of email in the context.